tests/script-based-pre/playback_print/print_vars.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// compile-flags: --edition 2021
+// kani-flags: playback
```

Comment:
I don't think these are being used, but if you wanted to leave them for documentation purposes we should add `-Z concrete-playback` to `kani-flags`.